ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Basics.hs

```diff
@@ -744,6 +744,12 @@ reapplyTrackingTicked after before =
     zipOverLedgerTablesTicked rawReapplyTracking after
   $ projectLedgerTables before
 
+rawTranslateDiff ::
+     (v -> v')
+  -> DiffMK k v
+  -> DiffMK k v'
+rawTranslateDiff f (ApplyDiffMK d) = ApplyDiffMK $ fmap f d
```

Comment:
This name confuses me as we talked about `Translate` but this is not related to the `Translate` operation is it?

Reply:
I then saw that this is used when defining `translateLedgerTablesWith` in era transitions `NP`s

Reply:
Can't we define a `Functor (DiffMK k)` instance?

Reply:
> Can't we define a `Functor (DiffMK k)` instance?

Will do

> This name confuses me ...

I added this function with the aim of having self-documenting code: I wanted to make explicit that mapping `(v -> v')` over a diff is an era translation. However, if it's confusing I'll remove it of course.